relay-event-schema/src/protocol/span.rs

```diff
@@ -257,6 +257,10 @@ pub struct SpanData {
     #[metastructure(field = "thread.name")]
     pub thread_name: Annotated<Value>,
 
+    /// Origin Transaction name of the span.
+    /// For INP spans, this is the route name where the interaction occurred.
+    pub transaction: Annotated<String>,
```

Comment:
Typing this as a `String` makes parsing fail if users put something else in there, but the normalization functions expect an `Annotated<String>`.